<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugControlUi</class>
 <widget class="QWidget" name="DebugControlUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Forzar estado en los DUTs</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>INACTIVO</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ESPERE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LISTO</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>REPETIDO</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SALTA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PASA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NO PASA</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Test repetitivo</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="7" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_Start">
          <property name="text">
           <string>Iniciar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Stop">
          <property name="text">
           <string>Parar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBox_DialogYes">
        <property name="text">
         <string>Responde SI a los cuadros de diálogo</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Número de iteraciones</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBox_Iterations">
        <property name="maximum">
         <number>99999</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Número de serie</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit">
        <property name="maxLength">
         <number>11</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBox_IncrementalSerial">
        <property name="text">
         <string>Número de serie incremental</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Iteración actual</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_currentIteration">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Contador de errores</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_ErrorsCounter">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <action name="actionStart">
   <property name="text">
    <string>Iniciar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
